bambi/plots/plot_cap.py

```diff
@@ -194,22 +197,30 @@ def plot_cap(
     cap_data = create_cap_data(model, covariates)
     idata = model.predict(idata, data=cap_data, inplace=False)
 
-    if hdi_prob is None:
-        hdi_prob = az.rcParams["stats.hdi_prob"]
-
-    if not 0 < hdi_prob < 1:
-        raise ValueError(f"'hdi_prob' must be greater than 0 and smaller than 1. It is {hdi_prob}.")
-
```

Comment:
This was here so we avoid computations when we raise an error.  Now I think it should be even before 

```python
cap_data = create_cap_data(model, covariates)
```